src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "Life_Pool";
+    private static final String DEFAULT_PASSWORD = "qwerty123";
+    private static final String NON_VALID_PASSWORD = "short";
+    private static final String EMPTY_FIELD = "";
+    private static final int DEFAULT_AGE = 22;
+    private static final int NON_VALID_AGE = 14;
```

Comment:
for testing it is better to take marginally unacceptable input data, in this case 17 years